src/main/frontend/fs.cljs

```diff
@@ -136,13 +136,13 @@
   [format s1 s2]
   (when (and s1 s2)
     (let [diff-result (diff/diff-words s1 s2)
-          block-pattern (config/get-block-pattern format)]
+          block-pattern (or (config/get-block-pattern format) "#")]
```

Comment:
`config/get-block-pattern` will return the default value.

Reply:
ah, it seems that it's redundant because the above line just called it. I'll make changes directly in the master branch.

Reply:
Also, it's not used anymore, I'll delete this function.